source/2019-09-25-ember-3-13-released.md

```diff
@@ -27,22 +27,22 @@ The Ember tutorial has already been completely rewritten for Octane, and the eas
 
 You can try out the Octane preview by disabling legacy behavior and specifying the Octane edition in `.ember-cli.js`.
 
+To opt in to the Octane preview:
+
 ```bash
-ember feature:disable jquery-integration
-ember feature:enable template-only-glimmer-components
-ember feature:disable application-template-wrapper
+npx @ember/octanify
 ```
 
-If you need more information on how to migrate away from these legacy features, check out the [Octane release plan](https://blog.emberjs.com/2019/08/15/octane-release-plan.html) blog post.
-
-To opt in to the Octane preview, add the following lines to the beginning of `.ember-cli.js`.
+This command does a number of things for you automatically:
 
-```js
-// .ember-cli.js
-const { setEdition } = require('@ember/edition-utils');
+* Migrates your `.ember-cli` file to `.ember-cli.js`, and opts in to Octane via `setEdition('octane')` (which is provided by `@ember/edition-utils`)
+* Ensures you have a new enough version of the `@ember/optional-features` addon installed
+* Updates your set of optional features to:
+  * disable [application-template-wrapper](https://guides.emberjs.com/release/configuring-ember/optional-features/#toc_application-template-wrapper)
+  * disable [jquery-integration](https://guides.emberjs.com/release/configuring-ember/optional-features/#toc_jquery-integration)
+  * enable [template-only-glimmer-components](https://guides.emberjs.com/release/configuring-ember/optional-features/#toc_template-only-glimmer-components)
```

Comment:
@rwjblue @CodingItWrong @chancancode Wanted to check in on getting this merged? Let me know, I'm same handle on Discord! 

✅ emberjs/ember-octanify#2 was merged

BTW the `npx` command also added `"default-async-observers": true` for me? I gave it a spin on https://github.com/ember-learn/ember-website repo.

cc @jenweber

Reply:
@amyrlam we would still need an `octanify` release before we'd be able to post this.

Since I already did another PR to manually include the instructions, and since this post is a few months old, it may make sense to just close this PR

Reply:
sounds good - that makes sense. will close, thanks @CodingItWrong!